cspell.json

```diff
@@ -1582,10 +1582,15 @@
     "nextamplifygen",
     "multirepo",
     "startbuild",
+    "PITR",
```

Comment:
nit: I see in this file that up to line 1448 all strings are ordered alphabetically, which it helps to easily identify if a word is already in the list or not. Although after that line there's no apparent order, I don't see a reason why not to keep the alphabetical order for new entries whenever it's possible.

Reply:
added in alphabetical order